src/components/VisualEditor.vue

```diff
@@ -26,6 +26,28 @@
 		<div class="editorSplitter">
 			<DiscoveryToolbar v-if="(showDiscoveryToolbar || isFullScreen) && editable" class="discoveryToolbar" :onAddProcess="insertProcess" />
 			<div class="graphBuilder" @drop="onDrop" @dragover="allowDrop">
+				<div v-if="showHelpOverlay" class="model-overlay">
+					<h2>Welcome!</h2>
+					<p>What you are seeing in this area of the {{ $config.appName }} is the visual model builder.</p>
+					<p>You can start building your model by dragging collections, processes etc. from the left area and dropping them here.</p>
+					<p>Alternatively, you can also import existing processes into the model builder:</p>
+					<ul>
+						<li>Paste the JSON from your clipboard by clicking <button type="button" @click="paste" title="Paste from clipboard"><i class="fas fa-paste"></i></button> or use <kbd>STRG</kbd> + <kbd>V</kbd> (Windows, Linux) or <kbd>⌘</kbd> + <kbd>V</kbd> (MacOS) when the model builder is in focus.</li>
```

Comment:
As mentioned in the comment, I'd just got with "CTRL" here.

Reply:
```suggestion
						<li>Paste the JSON from your clipboard by clicking <button type="button" @click="paste" title="Paste from clipboard"><i class="fas fa-paste"></i></button> or use <kbd>CTRL</kbd> + <kbd>V</kbd> (Windows, Linux) or <kbd>⌘</kbd> + <kbd>V</kbd> (MacOS) when the model builder is in focus.</li>
```